corehq/apps/users/forms.py

```diff
@@ -547,6 +548,14 @@ class NewMobileWorkerForm(forms.Form):
             </span>
         """
     )
+    send_account_confirmation_email = forms.BooleanField(
+        label=ugettext_noop("Send Account Confirmation Email Now?"),
+        help_text=ugettext_noop(
+            "The user will be sent their account confirmation email now. "
+            "Otherwise it must be sent manually at a later time."
```

Comment:
I don't think it's clear what will be required of the user if they uncheck this box. Is manually sending the email something the user would do in HQ?

Reply:
Yeah, that feature doesn't exist yet, but I'm working on it next. I'll update the language when it's more clear.